internal/tools/aws/client.go

```diff
@@ -185,16 +178,8 @@ func (c *Client) Service() *Service {
 			c.logger.WithError(err).Error("Can't load AWS Configuration")
 		}
 
-		// Load session for the V1 SDK
-		sess, err := NewAWSSessionWithLogger(c.config, c.logger.WithField("tools-aws", "client"))
```

Comment:
Just curious, did you notice any logging differences with how V2 is implemented? I can see they changed a few things here, but guessing logs look similar.

Reply:
I haven't seen much difference to be honest, though I usually run everything with `--debug --debug-helm`, so my logs are a bit noisy.